docs/concepts/service-catalog/index.md

```diff
@@ -0,0 +1,233 @@
+
+### Mutual TLS encryption
+
+The mutual TLS protocol encrypts communication between the Service Catalog extension API server and the main Kubernetes API server. 
+
+During installation, Service Catalog creates its own certificate authority (CA), and generates its own public and private keys, signed by this CA.
+The CA public certificate is installed into the main API server when Service Catalog registers its API.
+Service Catalog accesses the main API server CA certificate from the API server ConfigMap, after being granted the *extension-apiserver-authentication-reader* role. 
+
+Certificate rotation is handled by exposing a Service Catalog URL, similar to a `/statusz` endpoint, which returns the number of days until the CA and certificates expire.
+Alerts for certificate rotation can be created by monitoring this URL.
+
+### Authentication
+
+Service Catalog supports these methods of authentication: 
+
+* Basic (username/password)
+* [OAuth 2.0 Bearer Token](https://tools.ietf.org/html/rfc6750)
+
+## Usage
+
+The {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can use the Service Catalog API Resources to provision Managed Services and make them available within the Kubernetes cluster. The steps involved are:
+
+1. Listing the Managed Services available from a Service Broker.
+1. Provisioning a new instance of the Managed Service.
+1. Binding to the Managed Service, which returns the connection credentials.
+1. Mapping the connection credentials into the application.
+
+### Listing Managed Services
+
+First, the Cluster Operator must create a `ServiceBroker` resource within the `servicecatalog.k8s.io` group. This resource contains the URL and connection details necessary to access a Service Broker endpoint.
+
+This is an example of a `ServiceBroker` resource:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceBroker
+metadata:
+  name: cloud-broker
+spec:
+  # Points to the endpoint of a Service Broker. (This example is not a working URL.)
+  url:  https://servicebroker.somecloudprovider.com/v1alpha1/projects/service-catalog/brokers/default
+  # Describes the secret which contains the short-lived bearer token
+  authInfo:
+    bearer:
+      secretRef:
+        name: cloud-svc-account-secret
+        namespace: service-catalog
+```
+
+The following is a sequence diagram illustrating the steps involved in listing Managed Services available from a Service Broker:
+
+![List Services](/images/docs/service-catalog-list.svg){:height="80%" width="80%"}
+
+1. Once the `ServiceBroker` resource is added to Service Catalog, it triggers a *List Services* call to the external Service Broker.
+1. The Service Broker returns a list of available Managed Services, which is cached locally in a `ServiceClass` resource.
+1. The Cluster Operator can then get the list of available Managed Services using the following command:
```

Comment:
Is it just me, or is using The Definite Article for the specific user type a little weird? I would personally like to see "A cluster operator can" or "Cluster operators can". Let me know what you think.

Reply:
Yeah, I think you're right. I used "the" to be consistent with the rest of the canonical flow and treated it like a scenario. I can try "a" instead.